.github/workflows/preview.yml

```
@@ -0,0 +1,21 @@
+  pull-requests: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+      - run: yarn install
+      - run: yarn build:prod 
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)